tokio-console/src/config.rs

```diff
@@ -152,6 +177,9 @@ pub struct ColorToggles {
 #[derive(Debug, Clone, Deserialize, Serialize)]
 #[serde(deny_unknown_fields)]
 struct ConfigFile {
+    target_addr: Option<String>,
```

Comment:
oh, one last thought: it might be a bit clearer for users if we called the config key `default_target_addr` rather than `target_addr`, since a different target can always be connected to by overriding it from the command line?

but, on the other hand, all config keys are overridable, but we don't call the other ones `default_XXX`...

Reply:
Mmm...This is a tough one...Surely `default_target_addr` is clearer for users.
And I think might use `address` instead of `addr`.

Do you want to change `default_target_addr` or `default_target_address`?

Reply:
Either is fine, in my opinion; I think `addr` is a _relatively_ widespread abbreviation, but it's fine for the config file to be a bit more verbose than the CLI args, so `address` is also fine with me.

Reply:
Ok. I fix to use `default_target_addr`.

Reply:
great, thank you!